src/components/logTable/logTable.js

```diff
@@ -27,7 +27,7 @@ class LogTable extends React.Component {
       currItem: ""
     })
     // together JS running update
-    if (window.TogetherJS.running) {
+    if (!this.props.testEnv && window.TogetherJS.running) {
```

Comment:
Just curious, is this turning off togetherjs functionality for the test? Also random side note but do we need to test together js in our test coverage? or is that a dependency so it should be tested already and is good?

Reply:
Yes! It's a wacky way of testing this component. So if it's in testing env, we skip calling the togetherJS code. And yes! we assume TogetherJS has already been tested bc it's a dependency (I think they did test it)